scripts/_install_opencv_build_dependencies_on_macos.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+################################
+# Install OpenCV dependencies
+
+set -ex
+
+brew install pkg-config
```

Comment:
If we pivot to MacPorts, I can provide the equivalent for these libraries.

Reply:
Currently, I install these MacPorts on macOS for OpenCV building against MacPorts' Clang and LLVM 5.0 with OpenMP support:

```
sudo port install lapack eigen3-devel OpenBLAS-devel hdf5 py36-h5py-devel gdal py36-gdal freetype tesseract harfbuzz google-glog leptonica clang-5.0 llvm-5.0 libomp clang_select llvm_select

sudo port select clang mp-clang-5.0
sudo port select llvm mp-llvm-5.0
```